### DISCUSSION TITLE
关于解密

### DISCUSSION THREAD

**Discussion by Enigma-Soul** [Meloong-Git/PCL#2318] [提问] [answered]
龙猫能不能把那个解密改改，网上有人都发教程了{

2.回声洞提示“c。。。子”以及链接。。。bbs链接https://www.t/thread-。。。。。。11.html

3.根据帖子中的石墨：https://shi。。。mo...

4.回声洞提示“。，...的单元格找到短链https://d
5.短链跳转到链接https://pan.baidu.com/s/1PErf...
}上边删了一点（我有一个有趣且不难的解密过程，能不能改改，QQ私信行吗），别把这篇删了，**请让我和龙猫聊聊好吗**

**Comment by WForst-Breeze** [comment, marked as answer]
PCL2 已经不再限制讨论解密，并允许公开讨论，原因为即将更换隐藏主题系统（尽管好像到现在也不太有消息qwq

<details>
<summary>2021 年 8 月 20 日 PCL 1 群公告原文：</summary>

> 【主题功能调整预告】
在联机功能制作完成后，PCL2 计划重做大多数隐藏主题。这大概会导致大多数主题需要重新解锁……
因此，所有隐藏主题，包括极客蓝，从现在开始不再对讨论做出任何限制，你可以自由讨论解锁方法，或者不限人数地攻克解密。
极客蓝解密在获取主题后并没有结束……其剩余奖金将由目前完全完成了后续解密的 31 人平分。
主题功能也会在重做后对正式版开放使用，赞助解锁的功能将在此后改为启动器左上角的标题自定义。
如果你在主题功能重做前解锁了至少 7 个隐藏主题，在重做后会获得一份独特的绝版奖励！
</details>

**Reply by Enigma-Soul** [reply]
隐藏主题系统？

**Reply by Enigma-Soul** [reply]
我**
![image](https://github.com/Hex-Dragon/PCL2/assets/119654171/f8c1d335-58ed-4b22-9136-22e452ed3654)

**Reply by Enigma-Soul** [reply]
好不容易搞得

**Reply by WForst-Breeze** [reply]
麻烦请善用 Edit（

不用慌，这个我记得好久之前说就计划要弄了，直到现在也没具体开始重新制作，毕竟该做的新功能太多了，一个个来。个人觉得一时半会不会有变化的，尽管放心好了

**Reply by Enigma-Soul** [reply]
那个解密过程我觉得可以给龙猫说一下，让他引用一下“@龙猫”
